pkg/vm/stackitem/serialization.go

```diff
@@ -36,8 +36,8 @@ type SerializationContext struct {
 	seen         map[Item]sliceNoPointer
 }
 
-// deserContext is an internal deserialization context.
-type deserContext struct {
+// DeserializationContext is a deserialization context.
+type DeserializationContext struct {
 	*io.BinReader
```

Comment:
@roman-khimov, what do you think about converting `*io.BinReader` to unexported field? From the one hand, currently it's possible to check deserialization error directly from the context, from the other hand all other `*io.BinReader`'s methods (ReadB, ReadVarBytes, etc.) are exposed to the user which isn't good.